editor/js/ui/palette.js

```diff
@@ -267,9 +267,13 @@ RED.palette = (function() {
     function init() {
         $(".palette-spinner").show();
         if (RED.settings.paletteCategories) {
-            RED.settings.paletteCategories.forEach(createCategoryContainer);
+            RED.settings.paletteCategories.forEach(function(category){
+                createCategoryContainer(category, category); // TODO NLS enable categories from settings
```

Comment:
fyi, this can be handled by trying to lookup a message, but fall back to `category` if it doesn't exist:

    RED._("palette.core."+category,{defaultValue:category})

Will add that when I get this merged.